libraries/lib-sample-track/SampleTrack.h

```diff
@@ -16,6 +16,7 @@ Paul Licameli split from WaveTrack.h
 #include "SampleCount.h"
 #include "SampleFormat.h"
 #include "WideSampleSequence.h"
+#include "SampleTrackStuff.h"
```

Comment:
Remove this unnecessary line; then, this revision of the pull request makes no changes in `lib-sample-track`, which is very good.